docs/manage/ml/_index.md

```diff
@@ -5,20 +5,36 @@ weight: 40
 type: "docs"
 tags: ["data management", "data", "services"]
 no_list: true
-description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots"
+description: "Use Viam's built-in machine learning capabilities to train image classification models and deploy these models to your robots."
 # SME: Aaron Casas
 ---
 
-You can use Viam's built-in machine learning capabilities to [train image classification models](train-model/) and [deploy these models to your robots](../../services/ml/).
-You can also [upload and use existing models](upload-model/).
+Viam includes a built-in [machine learning (ML) service](/services/ml/) which provides your robot with the ability to learn from data and adjust its behavior based on insights gathered from that data.
+Common use cases include object detection, image classification, natural language processing, and speech recognition and synthesis, but your robot can make use of machine learning with nearly any kind of data.
 
-To capture and synchronize data to the platform, see [Data Management Service](../../services/data/).
-To view or export captured data, see [Data Management](../data/).
+Viam natively supports [TensorFlow Lite](https://www.tensorflow.org/lite) ML models as long as your models adhere to the [model requirements](/services/ml/#tflite_cpu-limitations).
+
+You can [train your own image classification models](/manage/ml/train-model/) or [add an existing model](/manage/ml/upload-model/) for object detection and classification within the platform using data from the [Data Management Service](../../services/data/).
+Object detection and classification models are commonly used to enable robots to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+
+To make use of ML models with your robot, you can use the built-in [ML model service](/services/ml/) to deploy and run the model.
+
+### Object detection and classification
```

Comment:
The second bullet point isn't about object detection and classification so I think best to remove

```suggestion
```